### PR TITLE
Add new contact validate

### PR DESCRIPTION
**Pull request by ericmoraess**
Validate a contact before save and show the wrongs fields.

### CODE REVIEW

File: AgendaUWP/Service/Model/Contact.cs [hunk 1]
```diff
@@ -68,6 +70,49 @@ public override string ToString()
...
+            if (messages.Count > 0)
+                throw new Exception(string.Join(Environment.NewLine, messages));
+        }
+
+        private bool IsValidPhone(string phone)
+        {
+            bool isValid = true;
+            try
+            {
+                System.Convert.ToInt32(phone);
```
Comment (soutolucas) [inline AgendaUWP/Service/Model/Contact.cs]: Please, use the method int.TryParse.

**Review by soutolucas — CHANGES_REQUESTED** [review]

File: AgendaUWP/Service/Model/Contact.cs [hunk 2]
```diff
@@ -68,6 +70,49 @@ public override string ToString()
         {
             return FullName + NickName + Age + Phone;
         }
+
+        public void Validate()
+        {
+            var messages = new List<string>();
+
+            if (IsNullOREmpty(_fullname))
```
Comment (lhohenrique) [inline AgendaUWP/Service/Model/Contact.cs]: You can use the string.IsNullOrEmpty method instead of creating a new one.

**Review by lhohenrique — CHANGES_REQUESTED** [review]

File: AgendaUWP/Service/Model/Contact.cs (hunk 1 above)
Comment (ericmoraess) [reply AgendaUWP/Service/Model/Contact.cs]: Ok! Done!

File: AgendaUWP/Service/Model/Contact.cs (hunk 2 above)
Comment (ericmoraess) [reply AgendaUWP/Service/Model/Contact.cs]: Cool! Done!

**Review by lhohenrique — APPROVED** [review]

**Review by soutolucas — APPROVED** [review]

**Pull request merged**